### PR TITLE
(CDAP-5684) Don’t stop Worker in the test.

### PR DESCRIPTION
**Pull request by chtyim**
Also speed up client tests by
making most client tests not to have explore enabled.

### CODE REVIEW

File: cdap-client-tests/src/test/java/co/cask/cdap/client/QueryClientTest.java [hunk 1]
```diff
@@ -43,15 +45,26 @@
  * Test for {@link QueryClient}.
  */
 @Category(XSlowTests.class)
-public class QueryClientTestRun extends ClientTestBase {
-  private static final Logger LOG = LoggerFactory.getLogger(QueryClientTestRun.class);
+public class QueryClientTest extends AbstractClientTest {
+
+  @ClassRule
+  public static final SingletonExternalResource STANDALONE = new SingletonExternalResource(
+    new StandaloneTester("explore.enabled", true));
```
Comment (anwar6953) [inline cdap-client-tests/src/test/java/co/cask/cdap/client/QueryClientTest.java]: use Constants.Explore.EXPLORE_ENABLED?

Comment (chtyim) [inline cdap-client-tests/src/test/java/co/cask/cdap/client/QueryClientTest.java]: Oh we have that? ok.

**Comment by anwar6953** [comment]
remainder LGTM

**Comment by chtyim** [comment]
Building: http://builds.cask.co/browse/CDAP-DUT4154

**Comment by chtyim** [comment]
Build passed.

**Pull request merged**